cmd/deploy/validate.go

```diff
@@ -21,7 +21,7 @@ import (
 
 func validateOptionVars(variables []string) error {
 	for _, v := range variables {
-		kv := strings.SplitN(v, "=", 2)
```

Comment:
I don't think we should do this. What if the value of the variable is a base 64 encoded valued with an '=' like `"STR='dGVzdA=='"`. As it is now it will correctly match variable `STR` variable value `dGVzdA==`.

With your changes, it would return an array of 4 elements and it would fail. Base64 encoded values are typical and they contains ' ='. 

We should just consider the value everything that goes after the first `=`.

I think it might be a good idea to include a test with the example I'm given above